<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiDialog_preferences</class>
 <widget class="QDialog" name="uiDialog_preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monal Log Viewer | preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="uiTab_color">
      <attribute name="title">
       <string>&amp;Colors</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QGridLayout" name="uiGridLayout_colorTab"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Left click on a color button will open the color picker, right click will delete this color entry.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="uiTab_misc">
      <attribute name="title">
       <string>&amp;Misc</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="uiGridLayout_miscTab"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="uiTab_formatTabs">
      <attribute name="title">
       <string>&amp;Log formatters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="uiVLayout_formatTabs"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="uiTab_history">
      <attribute name="title">
       <string>&amp;History</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="uiVLayout_historyTab"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Discard|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
